extension/llm/runner/multimodal_input.h

```diff
@@ -31,7 +31,7 @@ class ET_EXPERIMENTAL MultimodalInput {
   /// Type of multimodal input data
   enum class Type {
     TEXT, ///< Text string input
-    TOKENS, ///< Pre-tokenized input (vector of token IDs)
+    TOKENS, ///< Tokenizer encoded input (vector of token IDs)
```

Comment:
To be clear these are always text tokens right? Since image and audio just become embeddings?

Reply:
yeah